### DISCUSSION TITLE
模型效果

### DISCUSSION THREAD

**Discussion by sunmengnan** [Nicholasli1995/EvoSkeleton#65] [Q&A] [answered]
你好，大佬。
你提供的预训练模型是stage2层的，可以提供一下你训的3层的模型吗？我想试试效果能好多少，感谢🙇‍

**Comment by sunmengnan** [comment]
这个模型训练是S1-S11都用了吗？

**Comment by Nicholasli1995** [comment, marked as answer]
H36M和3DHP的效果已经有预训练模型了可以试了。训练in-the-wild三级模型也只用S15678，但要更多视角尺寸组合，更长生成时间。最近没有时间维护。以后有空考虑传一个更大的生成数据集。

**Reply by sunmengnan** [reply]
有用3DHP训的或者3DHP+H36M数据训的模型吗？ 提供用的只是H36M的吧

**Reply by Nicholasli1995** [reply]
> 有用3DHP训的或者3DHP+H36M数据训的模型吗？ 提供用的只是H36M的吧

目前只有用H36M训练的模型。如果想加入3DHP的数据训练可以准备3DHP的2D-3D点对再使用相同的方式训练。

**Reply by sunmengnan** [reply]
好的，想请教一下，如何用更多视角尺寸组合，生成大数据集？谢谢

**Reply by Nicholasli1995** [reply]
> 好的，想请教一下，如何用更多视角尺寸组合，生成大数据集？谢谢

视角是通过采样更多的相机位置和朝向，改变相机的外参即可。然后修改人体的尺寸，调整遗传算法的变异量参数，调整遗传算法的代数。这样做对训练内存和时间要求比较高。你的使用目标和场景是什么？最好根据你自己的使用场景来调整相机参数和其他参数，这样比盲目训练大模型更好。
